continuum/transforms/custom.py

```diff
@@ -0,0 +1,88 @@
+from typing import Tuple, Union
```

Comment:
Can you rename this module to a more explicit name, like `bg_swap.py`?

And eventually in the `transforms` package `__init__.py` import it so we can do `from continuum.transforms import BackgroundSwap`.